frontend/src/App.vue

```diff
@@ -9,7 +9,11 @@
     </div>
   </div>
   <feedback></feedback>
-  <footer class="my-8 text-lg text-center text-teal-light">Powered by Beamer</footer>
+  <imprint-modal></imprint-modal>
+  <footer class="my-8 text-lg text-center text-teal-light">
+    Powered by Beamer .
```

Comment:
Is it intentional that the `.` is here, also with space before it?

Reply:
Yes! At least that is what I have understood from @olympe-silve yesterday

Reply:
@olympe-silve could you please clarify whether this is intended?